docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,30 @@
+# Troubleshooting
+
+If you are having problems with your AmpliPi device start here
+
+## Audio
+#### Noise
+Audio noise can have a lot of causes, below are some common reasons:
+* **Mixing of the grounds**: AmpliPi has two different grounded lines, analog and digital. The most common way to mix them is by plugging in an audio device and attempting to power it by the usb on the rear of the unit, all usb-powered devices must be powered from a wall outlet
```

Comment:
maybe just say "mixing grounds" or "connecting grounds", "mixing of the grounds" sounds weird